devices/ePaper/Primitives/Color.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) 2022 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+
+namespace Iot.Device.ePaper.Primitives
+{
+    /// <summary>
+    /// A color data structure for RGB colors.
+    /// </summary>
+    public struct Color
+    {
+        /// <summary>
+        /// Gets the red value of this color.
+        /// </summary>
+        public byte R { get; }
+
+        /// <summary>
+        /// Gets the green value of this color.
+        /// </summary>
+        public byte G { get; }
+
+        /// <summary>
+        /// Gets the blue value of this color.
+        /// </summary>
+        public byte B { get; }
+
+        /// <summary>
+        /// Gets a value representing this color as On/Off.
+        /// </summary>
+        public byte Color1bpp => (byte)((R <= 0 && G <= 0 && B <= 0) ? 0x00 : 0xff);
```

Comment:
this may miss from the System.Drawing implemetation but you can add as extention. Same for the next functions